stdlib/LinearAlgebra/src/symmetric.jl

```diff
@@ -496,7 +496,7 @@ end
     if alpha isa Union{Bool,T} && beta isa Union{Bool,T}
         return BLAS.symv!(A.uplo, alpha, A.data, x, beta, y)
     else
-        return generic_matvecmul!(y, 'N', A, x, MulAddMul(α, β))
+        return @stable_muladdmul generic_matvecmul!(y, 'N', A, x, MulAddMul(α, β))
```

Comment:
I didn't bench it locally, but I think `generic_matvecmul!` might be as heavy as `generic_matmatmul!` for compiling? Should be good to re-check that the added `@stable_muladdmul` is OK.

Reply:
It does increase the compile time, but once compiled  `generic_matvecmul!()` does not allocate and is comparably fast to the BLAS version for dense matrices/vectors. Maybe the extra compile time is worth it here? Compilation of a single version of `generic_matvecmul!()` seems to take about 0.05-0.06s on my intel mbp, so in the non-const case it takes about 4x that to compile 5-arg `mul!()`.